test/units/testsuite-80.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Please merge the test with TEST-74, i.e. rename `testsuite-80.sh` -> `testsuite-74.machinectl.sh`.